spanner/cloud-client/pom.xml

```diff
@@ -63,6 +63,14 @@ limitations under the License.
           <groupId>io.opencensus</groupId>
           <artifactId>opencensus-api</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
please add  TODO comment here when we can remove this exclusion rule.

Reply:
I think these exclusions could live here forever. Neither samples, nor tests use io.grpc libraries directly, so if it works without those libraries, no need to include extra dependencies.